### PR TITLE
React to hosting changes

### PR DESCRIPTION
**Pull request by HaoK**
-AddHosting/RequestContainer/UseServices are gone
-Commenting out Startup tests for now (https://github.com/aspnet/Identity/issues/411)

cc @davidfowl @divega @suhasj

### CODE REVIEW

**Comment by divega** [comment]
:shipit:

File: test/Microsoft.AspNet.Identity.EntityFramework.Test/UserStoreTest.cs [hunk 1]
```diff
@@ -40,65 +42,72 @@ public void DropDb()
...
-            Assert.NotNull(userStore);
-            Assert.NotNull(userManager);
-
-            const string userName = "admin";
-            const string password = "a";
-            var user = new ApplicationUser { UserName = userName };
-            IdentityResultAssert.IsSuccess(await userManager.CreateAsync(user, password));
-            IdentityResultAssert.IsSuccess(await userManager.DeleteAsync(user));
-        }
+        //[Fact]
```
Comment (davidfowl) [inline test/Microsoft.AspNet.Identity.EntityFramework.Test/UserStoreTest.cs]: Is this dead code?

Comment (divega) [inline test/Microsoft.AspNet.Identity.EntityFramework.Test/UserStoreTest.cs]: See #411.

**Comment by HaoK** [comment]
268af34244b9987be7e5cedf371f495b3fa1feb1

**Pull request closed without merging**